docs/dev-guide/integration/rest-api/tasks.rst

```diff
@@ -59,6 +59,24 @@ Example Task Report::
  }
 
 
+
+.. _task_group_summary:
+
+Task Group Summary
```

Comment:
I think it would be more readable if we would create a separate section for task group and name it 'Task Group Management' and under this section include 'Task Group summary' and the 'Polling Task Groups Progress'